lib/khmer.hh

```diff
@@ -54,7 +54,7 @@ private:\
 #   define CIRCUM_MAX_VOL 200	// @CTB remove
 
 #   define SAVED_SIGNATURE "OXLI"
-#   define SAVED_FORMAT_VERSION 4
+#   define SAVED_FORMAT_VERSION 5
```

Comment:
On Tue, Jul 28, 2015 at 02:37:13PM -0700, Michael R. Crusoe wrote:
> > @@ -54,7 +54,7 @@ private:\
> >  #   define CIRCUM_MAX_VOL 200	// @CTB remove
> >  
> >  #   define SAVED_SIGNATURE "OXLI"
> > -#   define SAVED_FORMAT_VERSION 4
> > +#   define SAVED_FORMAT_VERSION 5
> 
> Gah, do we need per-file-type versioning?

Yeah, I kinda cringed. Maybe we could do *per release* file versioning,
so this would only be incremented after a release?